source/integrations-guide/integrations-guide-index.rst

```diff
@@ -10,87 +10,351 @@ Integrations Guide
+
+Learn more about what popular pre-built integrations are available and how to install them `here <https://developers.mattermost.com/integrate/plugins/>`_.
+
+
+
+.. image:: ../images/prebuilt-integrations.png
+  :alt: Pre-built plugins available for no-code integration
+  :width: 800
+
+.. note::
+The `Mattermost Marketplace <https://mattermost.com/marketplace/>`_ offers an expanded selection of community supported integrations. 
+
+
+Custom-built plugins
+^^^^^^^^^^^^^^^^^^^^
+
+**Technical complexity: Pro-code**
+
+.. include:: ../_static/badges/selfhosted-only.rst
+  :start-after: :nosearch:
+
+
+Custom built plugins are the most comprehensive way to add new features and customization to self-hosted Mattermost deployments. Custom plugins are ideal for customers wanting to change the behavior of the Mattermost server, desktop, and web apps without forking the core codebase to suit their organization’s needs.
+
+Building a custom plugin is a **software development** task, using ``Go`` for the server-side functionality and optionally ``TypeScript/React`` for UI components. Developers should be comfortable with Git, modern build tooling, and the `Mattermost Plugin API <https://developers.mattermost.com/integrate/reference/server/server-reference/>`_, including lifecycle hooks, KV storage, slash commands, and interactivity. Knowledge of testing, logging, and security best practices is essential for production-ready plugins, along with experience packaging and deploying plugins through the System Console or CLI. For teams without these skills, simpler options like webhooks, slash commands, or no-code workflow tools may be more practical.
+
+Plugins can authenticate and interact with Mattermost through `bot accounts <https://developers.mattermost.com/integrate/reference/bot-accounts/>`_, utilizing the `RESTful API <https://developers.mattermost.com/api-documentation/>`_.
+
 
-A :doc:`slash command </integrations-guide/run-slash-commands>` is similar to an :doc:`outgoing webhooks </integrations-guide/outgoing-webhooks/>`, but instead of listening to a channel, it's used as a command tool in a channel. The Mattermost slash command format is compatible with Slack's format, so you can easily port commands from Slack.
+Learn more about building your own plugin `here <https://developers.mattermost.com/integrate/plugins/>`_.
```

Comment:
Please avoid generic link text such as "here" and reference the page or section title instead.